File: tests/human/x/rb/two-sum.rb
def two_sum(nums, target)
  n = nums.length
  (0...n).each do |i|
    (i+1...n).each do |j|
      return [i, j] if nums[i] + nums[j] == target
    end
  end
  [-1, -1]
end

result = two_sum([2,7,11,15], 9)
puts result[0]
puts result[1]

File: tests/human/x/lua/two-sum.lua
local function twoSum(nums, target)
  local n = #nums
  for i=1,n do
    for j=i+1,n do
      if nums[i] + nums[j] == target then
        return {i-1, j-1}
      end
    end
  end
  return {-1,-1}
end
local result = twoSum({2,7,11,15}, 9)
print(result[1])
print(result[2])
